### PR TITLE
[MODULAR] Lopland Armor Kits (stolen from Splurt?)

### PR DESCRIPTION
**Pull request by delingar**
## About The Pull Request

Adding new security item - Lopland armor kit.
It can be used to upgrade any suit and head slot clothing to sec armor vest level. You can also wear guns after upgrade.
**DOES NOT WORK ON MODSUIT ELEMENTS, ARMOR VESTS AND HELMETS**

You can get kits in SecDrobe vending machine (20 items per restock unit) for 50 cr
Also, all department guards get their own pair of armor kits and one pair of kits is placed in Blueshield garment bag

Adding new opfor clothing bundle - MI-16 Infiltrator.
Contains 2 syndie armor kits, advanced chameleon headset, combat gloves, boots and sunglasses

## How This Contributes To The Nova Sector Roleplay Experience

Security and Guard players can make more variable suit loadouts

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  

https://github.com/NovaSector/NovaSector/assets/93310496/3058d28d-450b-43ad-9a26-fbe28c5e4a5e

https://github.com/NovaSector/NovaSector/assets/93310496/05d5664a-f5f5-4139-8f64-5a83b337fb7f

</details>

## Changelog

:cl:
add: Added armor kits for security, guards and blueshield. You can use them to upgrade almost any suit- and head-slot clothing to make them like security armor vest. Traitors can buy it in space suit category
add: Added new clothing bundle in opfor for really stealthy things.
/:cl:

### CODE REVIEW

**Comment by HoodMilk** [comment]
sec and department guard drip will go crazy

**Comment by SomeRandomOwl** [comment]
@delingar i was uh...about to do a review with how to fix the use_delay, its a simple fix based on the looks of things, like super simple

**Comment by delingar** [comment]
> @delingar i was uh...about to do a review with how to fix the use_delay, its a simple fix based on the looks of things, like super simple

I'm newbie at SS coding so, uh... I'm not really can to understand it rn

**Comment by Nerev4r** [comment]
Add it to cargo.

File: modular_nova/modules/traitor-uplinks/code/categories/suits.dm [hunk 1]
```diff
@@ -21,3 +21,11 @@
 	cost = 2
 	illegal_tech = FALSE
 	progression_minimum = 10 MINUTES
+
+/datum/uplink_item/suits/armorkit_traitor
```
Comment (RatFromTheJungle) [inline modular_nova/modules/traitor-uplinks/code/categories/suits.dm:25]: this is a no-go, we don't need 'crew item, but inexplicably better' appended to the uplink, especially as it gives no indicator at all of use

Comment (delingar) [reply modular_nova/modules/traitor-uplinks/code/categories/suits.dm:25]: Okay, made it. Traitors get security version

Comment (delingar) [reply modular_nova/modules/traitor-uplinks/code/categories/suits.dm:25]: Oh, shit, i forgot to fix opfor bundle, i'll fix it tomorrow (or you can do this on your own) cuz i'm going to bed (I'm European)

**Comment by delingar** [comment]
> Add it to cargo. 

Technically it's added already at sec wardrobe restock unit. + There're *20* kits per SecDrobe restock unit

**Comment by nikothedude** [comment]
against this for the reason of visual identifcation is useful

no longer will i be able to determine how threatening a guy is because that assistant over there, in the winter coat? yeah turns out he has a armor vest and helmet and i just woefully misjudged the fight

**Comment by GoldenAlpharex** [comment]
I don't think that adding essentially invisible armor is a good idea, nor should it be something that we should be aiming towards in the future. This isn't to discourage you from making any further contributions, I haven't looked at the code itself, but it's just the concept itself that I don't think fits well with the game.

We aim to improve the visibility of the effects of items in-game, to reduce the amount of reading that people need to do in order to figure out if someone is affected by something. Invisible armor would simply cause too much issues, simply because it would be impossible to tell that someone is more armored than they seem to be at a quick glance.

Instead, if you had something like, I don't know, a neck item that could deploy basically some sort of blue-ish forcefield on your sprite (using a filter and some more fancy stuff), that would give you armor equivalent to if you were wearing basic security armor, that you can turn on and off (so you can have drip, but also can fight when need be, which is when your sprite would have the overlay applied), then it would be a different story, but since you did mention that you're a new coder, I don't want to burden you with requested changes of that size. I'm also not saying that this idea I've literally just came up with is perfectly adequate for the game and would be merged without any question, I haven't taken a lot of time to put much thought into it, so we'd have to look into it to see if it's truly a good idea.

Anyway, for now, I think I'm going to close this PR, feel free to reach out on Discord in development-code if you want to discuss it some more.

**Pull request closed without merging**